.github/pull_request_template.md

```diff
@@ -1,15 +1,52 @@
-Describe your changes here.
+_Authors: Please fill out this form carefully and completely._
 
-Closes # (type the issue number after # if applicable; otherwise remove this line)
+_Reviewers: By approving this Pull Request you are approving the code change, as well as its deployment and mitigation plans._
+_Please read this description carefully. If you feel there is anything unclear or missing, please ask for updates._
+
+#### List the issues that this change affects.
+<!--Every code change must address _at least 1_ issue. Fixes a bug, completes a task, every change
+      should have a corresponding issue listed here. If one does not already exist, create one. -->
+      
+Closes # (type the GitHub issue number after #)
+
+### What are you trying to accomplish?
+<!-- Provide a description of the changes. -->
 
 ### Screenshots
+<!-- Provide before/after screenshots, videos, or graphs for any visual changes; otherwise, remove this section -->
```

Comment:
```suggestion
```

Reply:
Can you provide any more context for this suggestion? The instructions note that this section is fine to remove if the changes do not impact UI.

Reply:
I see now that you moved the screenshot suggestion to elsewhere in the template. That's fine with me; I'll push a recommended change to the instruction there but otherwise I'm good with this.